samples/nrf9160/pdn/src/main.c

```diff
@@ -69,6 +80,52 @@ void main(void)
 
 	printk("Default APN is %s\n", apn);
 
+	err = nrf_getifaddrs(&ifaddrs);
+	if (err) {
+		printk("nrf_getifaddrs() failed, err %d\n", err);
+		return;
+	}
+
+	ifaddr = ifaddrs;
```

Comment:
Let's move this to right before the `while` since it's part of the iteration pattern.